server/kolide/queries.go

```diff
@@ -64,3 +79,63 @@ type Query struct {
 	// table in the MySQL backend.
 	Packs []Pack `json:"packs" db:"-"`
 }
+
+const (
+	QuerySpecKind = "OsqueryQuery"
+)
+
+type queryYaml struct {
```

Comment:
YAML is just a serialization format. IMO we should come up with a better name for the configuration data than YAML.